kafka-ui-react-app/src/components/Brokers/BrokersList/BrokersList.tsx

```diff
@@ -12,17 +12,18 @@ import { ColumnDef } from '@tanstack/react-table';
 import { clusterBrokerPath } from 'lib/paths';
 import Tooltip from 'components/common/Tooltip/Tooltip';
 import ColoredCell from 'components/common/NewTable/ColoredCell';
+import { BrokerDiskUsage } from 'generated-sources';
 
 import SkewHeader from './SkewHeader/SkewHeader';
 import * as S from './BrokersList.styled';
 
-const NA = 'N/A';
+const NA = 'N/A' as unknown;
```

Comment:
if we are going to do type casting can't we do it all of it here ? otherwise it the half of it are someplace.